src/validation/fields/github-handle.validation.ts

```diff
@@ -1,7 +1,8 @@
 import { body } from 'express-validator';
 
 import { ErrorMessages } from '../error.messages';
+import { validateGithubHandles } from './helper/github-handles.validation.helper';
 
 export const githubHandleValidation = [
-    body('github_handle').not().isEmpty({ ignore_whitespace: true }).withMessage(ErrorMessages.GIT_HANDLE),
+    body('github_handle').not().isEmpty({ ignore_whitespace: true }).withMessage(ErrorMessages.GIT_HANDLE).custom(validateGithubHandles),
```

Comment:
If you submit the other page forms with invalid `github_handle` field, the error messages are now appearing twice. I think this is becuase of the custom validator throwing an error after the `withMessage(ErrorMessages.GIT_HANDLE)` code. Could you investigate to fix it?

<img width="696" alt="Screenshot 2024-03-18 at 11 35 58" src="https://github.com/cabinetoffice/github-requests-app/assets/79383398/c3061b33-118f-4927-bf54-33ade01ba581">